ArduCopter/mode.h

```diff
@@ -563,7 +563,8 @@ class ModeBrake : public Mode {
         Copter::Mode(copter)
         { }
 
-    bool init(bool ignore_checks) override;
+    bool ok_to_enter() const override;
```

Comment:
This has no implementation, should be removed.

Reply:
Will do.